Vienna/Sources/Main window/WebKitArticleTab.swift

```diff
@@ -30,18 +30,18 @@ class WebKitArticleTab: BrowserTab, ArticleContentView, CustomWKUIDelegate {
                 return
             }
 
-            let htmlPath = converter.prepareArticleDisplay(self.articles)
-
-            webView.loadFileURL(htmlPath, allowingReadAccessTo: htmlPath.deletingLastPathComponent())
-            DispatchQueue.main.asyncAfter(deadline: .now() + 0.2) {
-                do {
-                    try FileManager.default.removeItem(at: htmlPath)
-                } catch {
-                }
-            }
+            deleteHtmlFiles()
+
+            let articleFileName = articles.count >= 1 ? "\(articles[0].guid)+\(articles.count)" : "dummy"
```

Comment:
I like the principle, but you are susceptible to have conflicting GUIDs, especially with group folders and smart folders.

So I prefer to maintain an UUID approach.
This would also avoid to introduce an additional framework (SwiftSlug).